src/tests/e2e/receipt.test.e2e.js

```diff
@@ -0,0 +1,569 @@
+    async () => {
+      const transfer = transferDataList[1]
+
+      await page.goto(`${process.env.E2E_TEST_URL}/receipt?receivingId=${transfer.receivingId}`)
+      const loginPage = new LoginPage()
+
+      await Promise.all([
+        reduxTracker.waitFor(
+          [
+            {
+              action: {
+                type: 'GET_TRANSFER_FULFILLED'
+              }
+            }
+          ],
+          [
+            // should not have any errors
+            {
+              action: {
+                type: 'ENQUEUE_SNACKBAR',
+                notification: {
+                  options: {
+                    variant: 'error'
+                  }
+                }
+              }
+            }
+          ]
+        ),
+        page.waitForNavigation({
+          waitUntil: 'networkidle0'
+        }),
+        loginPage.login(
+          process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+          process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD,
+          true
+        )
+      ])
+
+      const receiptPage = new ReceiptPage()
+
+      const title = await receiptPage.getReceiptFormInfo('title')
+      expect(title).toEqual('Pending to Receive')
+
+      const sender = await receiptPage.getReceiptFormInfo('sender')
+      expect(sender.name).toEqual(transfer.senderName)
+      expect(sender.email).toEqual(transfer.sender)
+
+      const recipient = await receiptPage.getReceiptFormInfo('recipient')
+      expect(recipient.name).toEqual(transfer.receiverName)
+      expect(recipient.email).toEqual(transfer.destination)
+
+      const transferAmount = await receiptPage.getReceiptFormInfo('transferAmount')
+      expect(transferAmount.transferAmount).toEqual(transfer.transferAmount)
+      expect(transferAmount.currencyAmount).toEqual(transfer.transferFiatAmountSpot)
+      expect(transferAmount.symbol).toEqual(getCryptoSymbol(transfer.cryptoType))
+
+      expect(await receiptPage.getReceiptFormInfo('securityAnswer')).toBeNull()
+
+      const { message } = await receiptPage.getReceiptFormInfo('sendMessage')
+      expect(message).toEqual(transfer.sendMessage)
+
+      const { sendOn } = await receiptPage.getReceiptFormInfo('sendOn')
+      const expectTime = moment
+        .unix(transfer.senderToChainsfer.txTimestamp)
+        .format('MMM Do YYYY, HH:mm:ss')
+      expect(sendOn).toEqual(`Sent on ${expectTime}`)
+
+      const explorerPage = await getNewPopupPage(browser, async () => {
+        await receiptPage.dispatchActions('openSendExplorer')
+      })
+      expect(explorerPage.url()).toEqual(`https://rinkeby.etherscan.io/tx/${transfer.sendTxHash}`)
+      await explorerPage.close()
+
+      const { transferId } = await receiptPage.getReceiptFormInfo('transferId')
+      expect(transferId).toEqual(transfer.receivingId)
+    },
+    timeout
+  )
+
+  it(
+    `Receiving ID: ${transferDataList[2].receivingId} (SEND_DIRECT_TRANSFER_CONFIRMED) RECEIVER`,
```

Comment:
do we have direct transfer in mock data?

Reply:
typo. fixed